modin/pandas/test/test_series.py

```diff
@@ -1721,6 +1721,26 @@ def test_fillna(data):
     df_equals(modin_series.fillna(method="ffill"), pandas_series.fillna(method="ffill"))
     df_equals(modin_series.fillna(0, limit=1), pandas_series.fillna(0, limit=1))
 
+    series_data = pandas_series.tolist()
```

Comment:
Why didn't you add a test for `DataFrame`?

Reply:
I am not sure of `limit` doesn't have any affect for `DataFrame`. An example:
```python
import pandas
import numpy as np
df = pandas.DataFrame([[np.nan, 2, np.nan, 0],
    [3, 4, np.nan, 1],
    [np.nan, np.nan, np.nan, 5],
    [np.nan, 3, np.nan, 4]],
    columns=list('ABCD')
)
values = {'A': 0, 'B': 1, 'C': 2, 'D': 3}
df.fillna(value=values, limit=1)
     A    B    C  D
0  0.0  2.0  2.0  0
1  3.0  4.0  NaN  1
2  NaN  1.0  NaN  5
3  NaN  3.0  NaN  4
df.fillna(value=values, limit=2)
     A    B    C  D
0  0.0  2.0  2.0  0
1  3.0  4.0  2.0  1
2  0.0  1.0  NaN  5
3  NaN  3.0  NaN  4
```

Reply:
I wrote this incorrectly. `Limit` doesn't have effect in case of `DataFrame.fillna(value=DataFrame)`. When `value` is a `Series` or `dict` then `limit` has effect. I implemented tests for `limit` with `value` of different types and found a bug in my implementation for `value=Series`. It should be fixed now.